unittests/lit.cfg.py

```diff
@@ -0,0 +1,27 @@
+# -*- Python -*-
+
+# Configuration file for the 'lit' test runner.
+
+import os
+import lit.formats
+
+# name: The name of this test suite.
+config.name = 'Pstore-Unit'
```

Comment:
s/Pstore/pstore/

Reply:
Done, thought I'm not sure why that's preferred. "pstore-Unit" looks odd to me!

Reply:
pstore (all lower case) is the product name. It’s pstore, ”store” (somewhere to safely keep things) rather than Pstore, “pee store” (something altogether different).